npm2deb/__init__.py

```diff
@@ -53,6 +54,9 @@ def __init__(self, module_name=None, args={}):
                 self.debian_debhelper = args['debhelper']
             if 'noclean' in args:
                 self.noclean = args['noclean']
+            if 'no_registry' in args:
+                self.noregistry = args['no_registry']
```

Comment:
This variable : `self.noregistry` doesn't seem to be used anywhere in the PR to ascertain if the argument has been passed

Reply:
Yeah, I know.  I thought I had set that up, but apparently I forgot.